annotation-file-utilities/scripts/insert-annotations

```diff
@@ -28,8 +28,8 @@ while [ "$#" -gt 0 ]; do
 done
 
 AFU=${AFU:-$(dirname $0)/..}
-ANNOTATION_FILE_UTILS=${AFU}/bin:${AFU}/../scene-lib/bin:${AFU}/../asmx/bin:${AFU}/annotation-file-utilities-all.jar
-JAVAC_JAR=${JAVAC_JAR:-${ANNOTATION_FILE_UTILS}/lib/compiler-2.4.0.jar}
+ANNOTATION_FILE_UTILS=${AFU}/annotation-file-utilities-all.jar
+JAVAC_JAR=${JAVAC_JAR:-${AFU}/lib/compiler-2.4.0.jar}
```

Comment:
JAVAC_JAR was set to `${AFU}/bin:${AFU}/../scene-lib/bin:${AFU}/../asmx/bin:${AFU}/annotation-file-utilities-all.jar/lib/compiler-2.4.0.jar`.